### PR TITLE
Tr 2 check the cart

### PR DESCRIPTION
**Pull request by Obibus**

### CODE REVIEW

File: src/test/java/elements/ShopPageElements.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+import base.AbstractBasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ShopPageElements extends AbstractBasePage {
+    public ShopPageElements(WebDriver driver) {
+        super(driver);
+    }
+    //shop page
```
Comment (kimkaminska) [inline src/test/java/elements/ShopPageElements.java:11]: please, leave empty line

File: src/test/java/elements/ShopPageElements.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
+package elements;
+
+import base.AbstractBasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ShopPageElements extends AbstractBasePage {
+    public ShopPageElements(WebDriver driver) {
+        super(driver);
+    }
```
Comment (kimkaminska) [inline src/test/java/elements/ShopPageElements.java:10]: please, leave empty line after 10 line

File: src/test/java/tests/CheckCartTest.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    public static final String MY_ACCOUNT_PAGE_LINK = "https://epicentrk.ua/ua/personal/order/make/";
+    public static final String IPHONE = "iPhone 14 Pro Max";
+
+    @Test
+    public void checkStoreLocationSelection() {
+        HomePage homePage = new HomePage(driver);
+        CartPage cartPage = new CartPage(driver);
+        SoftAssert softAssert = new SoftAssert();
+        ShopPage shopPage = new ShopPage(driver);
+
```
Comment (DashaYab) [inline src/test/java/tests/CheckCartTest.java:20]: remove empty line please)

File: src/test/java/tests/CheckCartTest.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                .clickRemoveFromCartVerificationCloseBtn()
+                .clickDeleteAllProducts()
+                .clickNoButtonInVerificationDelete()
+                .clickMakePurchase();
+
+        String currentUrl = getCurrentUrl();
+        softAssert.assertTrue(currentUrl.equals(MY_ACCOUNT_PAGE_LINK));
+        softAssert.assertAll();
+    }
+
```
Comment (DashaYab) [inline src/test/java/tests/CheckCartTest.java:72]: remove empty line please

File: src/test/java/tests/CheckCartTest.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    public static final String IPHONE = "iPhone 14 Pro Max";
+
+    @Test
+    public void checkStoreLocationSelection() {
+        HomePage homePage = new HomePage(driver);
+        CartPage cartPage = new CartPage(driver);
+        SoftAssert softAssert = new SoftAssert();
+        ShopPage shopPage = new ShopPage(driver);
+
+
```
Comment (kimkaminska) [inline src/test/java/tests/CheckCartTest.java:21]: please, delete empty line

File: src/test/java/tests/CheckCartTest.java [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+        cartPage.
+                clickDeleteAllProducts()
+                .clickRemoveFromCartVerificationCloseBtn()
+                .clickDeleteAllProducts()
+                .clickNoButtonInVerificationDelete()
+                .clickMakePurchase();
+
+        String currentUrl = getCurrentUrl();
+        softAssert.assertTrue(currentUrl.equals(MY_ACCOUNT_PAGE_LINK));
```
Comment (kimkaminska) [inline src/test/java/tests/CheckCartTest.java:69]: Please, change softAssert using get.currentURl().contains("/personal/order/make") without using constant

File: src/test/java/tests/CheckCartTest.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (kimkaminska) [reply src/test/java/tests/CheckCartTest.java:72]: Please, add Steps and Descriptions

File: src/test/java/tests/CheckCartTest.java (hunk 5 above)
Comment (Obibus) [reply src/test/java/tests/CheckCartTest.java:21]: corrected

File: src/test/java/tests/CheckCartTest.java (hunk 3 above)
Comment (Obibus) [reply src/test/java/tests/CheckCartTest.java:20]: corrected

File: src/test/java/elements/ShopPageElements.java (hunk 2 above)
Comment (Obibus) [reply src/test/java/elements/ShopPageElements.java:10]: corrected

File: src/test/java/tests/CheckCartTest.java (hunk 4 above)
Comment (Obibus) [reply src/test/java/tests/CheckCartTest.java:72]: corrected

[the remark this answers is not in the document]
Comment (Obibus) [reply src/test/java/tests/CheckCartTest.java:72]: corrected

**Review by VadymTT — APPROVED** [review]

**Review by DashaYab — APPROVED** [review]

**Pull request merged**